public/sass/elements/forms/_form-multiple-choice.scss

```diff
@@ -43,6 +39,14 @@
 
   label {
     cursor: pointer;
+    padding: (8px $gutter-one-third 9px 12px);
```

Comment:
Something in my head is saying it's something to do with an old sass compiler which got upset without them. But that may be imagined nonsense. And if it were ever true, it's not true any more, it compiles fine without them so I'd support removing them yeah.